src/note-components/Navbar/navbar.jsx

```diff
@@ -1,13 +1,24 @@
+import { useAuth } from "../../Context/auth-context";
+import { useNavigate } from "react-router-dom";
 import "./navbar.css"
 const Navbar = () => {
+const { authDispatch } = useAuth();
+const navigate = useNavigate();
+
+const logoutHandler = () => {
+navigate("/")
+localStorage.removeItem("token");
+localStorage.removeItem("user");
+authDispatch({ type: "LOGOUT" });
```

Comment:
Just a personal opinion you do do all the task of local storage and authdispatch and then write the code of navigate() at the end.